tests/v2/08_full_workflow/test_full_workflow_local_processes_v2.py

```diff
@@ -0,0 +1,353 @@
+import json
+import shlex
+import subprocess
+import time
+from concurrent.futures.process import BrokenProcessPool
+from glob import glob
+from pathlib import Path
+
+import pytest
+from common_functions import _task_name_to_id
+from common_functions import failing_workflow_UnknownError
+from common_functions import full_workflow
+from common_functions import full_workflow_TaskExecutionError
+from common_functions import non_executable_task_command
+from common_functions import PREFIX
+from devtools import debug
+
+from fractal_server.app.runner.exceptions import JobExecutionError
+from fractal_server.app.runner.filenames import FILTERS_FILENAME
+from fractal_server.app.runner.filenames import HISTORY_FILENAME
+from fractal_server.app.runner.filenames import IMAGES_FILENAME
+from fractal_server.app.runner.filenames import WORKFLOW_LOG_FILENAME
+from fractal_server.app.runner.v2._local_processes.executor import (
+    FractalProcessPoolExecutor,
+)
+
+FRACTAL_RUNNER_BACKEND = "local_processes"
+
+
+async def test_full_workflow_local(
+    client,
+    MockCurrentUser,
+    tmp777_path,
+    project_factory_v2,
+    dataset_factory_v2,
+    workflow_factory_v2,
+    override_settings_factory,
+    tmp_path_factory,
+    fractal_tasks_mock,  # needed
+):
+    # Use a session-scoped FRACTAL_TASKS_DIR folder
+    override_settings_factory(
+        FRACTAL_RUNNER_BACKEND=FRACTAL_RUNNER_BACKEND,
+        FRACTAL_RUNNER_WORKING_BASE_DIR=tmp777_path / "artifacts",
+        FRACTAL_TASKS_DIR=tmp_path_factory.getbasetemp() / "FRACTAL_TASKS_DIR",
+        FRACTAL_LOCAL_CONFIG_FILE=tmp777_path / "local_config.json",
+    )
+
+    # Test invalid local config (LocalBackendConfigError)
```

Comment:
Let's move this block to a new test

Reply:
Let's have a new subfolder in tests/v2 (e.g. `XX_backends`), and add a test similar to the current `test_get_local_backend_config_fail`.